dask_ml/model_selection/_validate.py

```diff
@@ -0,0 +1,43 @@
+import functools
+
+from sklearn.model_selection import cross_validate as _cross_validate
+from sklearn.model_selection import cross_val_score as _cross_val_score
+from sklearn.model_selection import cross_val_predict as _cross_val_predict
+from dask.base import is_dask_collection
+
+from dask_ml import joblib
+
+
+def _joblibify(f):
+    """
+    Allows execution of functions using joblib('dask') as backend.
+
+    :param f: A function with signature ``f(something, X, y=None, **kwargs)``
+    :return: A function that runs on dask
+    """
+    @functools.wraps(f)
+    def wrap(*args, **kwargs):
+        X = args[1]
+        y = kwargs.get("y")
+
+        if not y and len(args) == 3:
+            y = args[2]
+
+        if not y:  # unsupervised
+            scatter = [X]
+        else:
+            scatter = [X, y]
+
+        if is_dask_collection(X) or is_dask_collection(y):
+            raise NotImplementedError('This method is not implemented '
```

Comment:
This should note that it isn't implemented for *dask collections* yet.

Perhaps you could indicate that users can pre-compute `X` and `y` if they fit in memory.

```
If your data fit in RAM on each worker, you can pre-compute the data with

X, y = dask.compute(X, y)

before calling {f.__name__}
```